pkg/controller/postgresql.go

```diff
@@ -457,3 +466,40 @@ func (c *Controller) postgresqlDelete(obj interface{}) {
 
 	c.queueClusterEvent(pg, nil, spec.EventDelete)
 }
+
+/*
+  Ensures the service account required by StatefulSets to create pods exists in a namespace before a PG cluster is created there so that a user does not have to deploy the account manually.
+
+  The operator does not sync these accounts after creation.
+*/
+func (c *Controller) createPodServiceAccounts(event spec.ClusterEvent) error {
+
+	namespace := event.NewSpec.GetNamespace()
+	podServiceAccountName := c.opConfig.PodServiceAccountName
+
+	if _, ok := namespacesWithPodServiceAccount.Load(namespace); ok {
+		c.logger.Infof(fmt.Sprintf("The required pod service account %v already exists in the namespace %v", podServiceAccountName, namespace))
```

Comment:
Showing this for every add, update or sync operation is a sure way to annoy the user.  I'd opt for now displaying anything here.